tst/components/ui/Modal/ModalButtonGroup.test.tsx

```diff
@@ -111,4 +111,24 @@ describe('ModalButtonGroup', () => {
     expect(component.props.value.onClose).toHaveBeenCalled();
     expect(getByLabelText('close')).toBeInTheDocument();
   });
+
+  it('does not pass the onClose function when "dismissible" is false', () => {
+    const onClose = jest.fn();
+    const labelID = 'test-label';
+    const context = { onClose, labelID, dismissible: false };
+    const component = (
+      <ModalContext.Provider value={context}>
+        <ModalButtonGroup
+          primaryButtons={[<ModalButton label="close" closesModal />]}
+        />
+      </ModalContext.Provider>
+    );
+    const { getByTestId } = renderWithTheme(
+      lightTheme,
+      component
+    );
+    const modalButton = getByTestId('button');
```

Comment:
Is this actually finding the button? I don't think it is.

Reply:
If we weren't finding the button, we'd get an error from this line, and we don't. Why do you think that it's not finding the button? The change I'm making will not remove the `ModalButtons` completely, it just doesn't inject the `oncClose` function to their click handler.

Reply:
My bad. I missed that this `data-testid` was coming all the way from our base button.